packages/client/package.json

```diff
@@ -5,23 +5,29 @@
   "description": "Package for connecting to any Penumbra extension, including Prax.",
   "type": "module",
   "scripts": {
-    "build": "tsc && vite build",
-    "clean": "rm -rfv dist",
-    "lint": "eslint \"**/*.ts*\""
+    "build": "tsc",
+    "clean": "rm -rfv dist package penumbra-zone-client-*.tgz",
+    "lint": "eslint \"**/*.ts*\"",
+    "prebuild": "$npm_execpath run clean",
+    "prepack": "$npm_execpath run build"
```

Comment:
What's `$npm_execpath` do here?

Reply:
`$npm_execpath` will refer to npm/pnpm/yarn/whatever. it seems like this is the only cross-package-manager way for one package script to call another. these scripts probably don't matter because a consumer won't call them outside of our monorepo, but in general when writing a package for publication it's better to not name an explicit npm/pnpm/yarn in scripts